src/actions/configuration/first-time-configuration-action.php

```diff
@@ -186,6 +215,16 @@ public function set_enable_tracking( $params ) {
 			$success = $this->options_helper->set( 'tracking', $params['tracking'] );
 		}
 
+		/**
+		 * Action: 'wpseo_post_update_enable_tracking' - Allows for Hiive event tracking.
+		 *
+		 * @param array The new value.
+		 * @param array The old value.
+		 * @param bool  Whether the option failed to be stored.
+		 */
+		// $success is negated to be aligned with the other two actions which pass $failures.
+		\do_action( 'wpseo_post_update_enable_tracking', $params['tracking'], $option_value, ! $success );
```

Comment:
* You used in the add_action `wpseo_ftc_post_update_enable_tracking` elsewhere!
* Maybe signal this as `@internal`? Or create documentation PR?

Reply:
Done.